variables.tf

```diff
@@ -2,6 +2,23 @@
 # Account Variables
 ########################################################################################################################
 
+variable "global_resource_group_name" {
+  type        = string
+  description = "The name of the global resource group to create, when this variable is provided only one resource group will be created and all other resource group name variables will be ignored."
```

Comment:
You should use a full stop instead of a comma here. This is 2 difference sentences.